Tasks/SshV0/ssh.ts

```diff
@@ -173,4 +174,14 @@ async function run() {
     }
 }
 
+function tryDeleteFile(filePath: string): void {
+    if (fs.existsSync(filePath)) {
+        try {
+            fs.unlinkSync(filePath);
+        } catch (err) {
+            console.error(err);
```

Comment:
Yeah, see the screenshot -- not this line but the other `console.error` I added is in there.

You also get an error to print when you fail the task, and that's what gives you the red text with the `[error]` line.  I was going to just do that in one place, but maybe it would be better to print all errors like that.

Reply:
Oh, there's a task lib `error` function that will do that too.

Reply:
There:
![image](https://user-images.githubusercontent.com/33549821/50007546-4cd8b480-ff7f-11e8-80d1-0d75d4cd1ae5.png)